applications/nrf5340_audio/src/bluetooth/bt_stream/broadcast/broadcast_source.h

```diff
@@ -62,7 +55,7 @@ int broadcast_source_enable(void);
 int broadcast_source_disable(void);
 
 /**
- * @brief	Enable the LE Audio broadcast source.
+ * @brief	Enable the LE Audio broadcast (BIS) source.
```

Comment:
Nice, but for consistency we should decide if we want to write this or:
```suggestion
 * @brief	Enable the Bluetooth LE Audio broadcast (BIS) source.
```

Reply:
Unicast server has "Bluetooth" included

Reply:
Yeah, I can never remember the 'correct' way to use. I see that we have a mix in our code base

Reply:
Should I fix this in my cleanup PR?